cartofacade/capi.go

```diff
@@ -239,15 +239,14 @@ func (vc *Carto) getPointCloudMap() ([]byte, error) {
 		return nil, err
 	}
 
+	pcd := bstringToByteSlice(value.point_cloud_pcd)
+
 	status = C.viam_carto_get_point_cloud_map_response_destroy(&value)
 	if err := toError(status); err != nil {
 		return nil, err
 	}
 
-	if value.point_cloud_pcd != nil {
-		return bstringToByteSlice(value.point_cloud_pcd), nil
-	}
-	return nil, errors.New("nil pointcloud")
+	return pcd, nil
```

Comment:
Go convention tells us that if the error we checked on line 238 is nil then we can safely assume that `pcd` is valid